tutorial.md

```diff
@@ -6,6 +6,7 @@ slug: rust-tutorial-markdown-compiler
 author: Jesse Lawson
 type: post
 tutorial: true
+hackernewslink: https://news.ycombinator.com/item?id=23206660
```

Comment:
Based on the feedback I've received from HN, I created a new doohickey here so that I can easily include the HN thread.